### PR TITLE
Finish it up

### PR DESCRIPTION
**Pull request by imbstack**
Pretty and such.

### CODE REVIEW

File: kraller/kraller.py [hunk 1]
```diff
@@ -19,19 +20,23 @@
 
 app = Flask(__name__)
 app.config.from_envvar('KRALLER_SETTINGS')
+app.secret_key = app.config['SECRET_KEY']
```
Comment (hashbrowncipher) [inline kraller/kraller.py:23]: I don't think we need this.  Will check.

Comment (imbstack) [inline kraller/kraller.py:23]: It yelled at me until I added one...

Something in itsdangerous uses it iirc.

File: kraller/kraller.py [hunk 2]
```diff
@@ -42,6 +47,11 @@ def decorated(*args, **kwargs):
             return redirect(cas_login_url())
     return decorated
 
+
+def in_blacklist(name):
+    return name in set(map(lambda x: x.strip(), open(app.config['BLACKLIST_FILE']).readlines()))
```
Comment (hashbrowncipher) [inline kraller/kraller.py]: Using set() here is unnecessary.  Construction of the set will take more time than O(n) lookup in a list.

File: kraller/kraller.py [hunk 3]
```diff
@@ -61,37 +71,52 @@ def login():
             return redirect(redirect_to)
     return abort(401)
 
+
+@app.route('/logout')
+def logout():
+    try:
+        session.pop('username')
```
Comment (hashbrowncipher) [inline kraller/kraller.py]: do you have something against del()?  No need to change it, I just find it weird.

Comment (imbstack) [inline kraller/kraller.py]: I just didn't think of it I guess.  I agree it's kinda ugly.

**Pull request merged**